### PR TITLE
Add a form that submits non-securely

### PR DESCRIPTION
**Pull request by ericlaw1979**
I'm doing some work to clean up our handling of "mixed" pages where the mixed content is a form that targets HTTP. PTAL?

### CODE REVIEW

File: domains/mixed/mixed-form/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (lgarron) [inline domains/mixed/mixed-form/index.html]: Could you make this file a copy of `/common/mixed/form/`, for consistency with all the other mixed inputs?

File: domains/misc/badssl.com/index.html [hunk 2]
```diff
@@ -56,6 +56,7 @@ <h2 id="mixed-content"><span class="emoji">🖼</span>Mixed Content</h2>
     <hr>
     <a href="https://mixed.{{ site.domain }}/" class="dubious"><span class="icon"></span>mixed</a>
     <a href="https://mixed-favicon.{{ site.domain }}/" class="dubious"><span class="icon"></span>mixed-favicon</a>
+    <a href="https://mixed-form.{{ site.domain }}/" class="dubious"><span class="icon"></span>mixed-form</a>
```
Comment (lgarron) [inline domains/misc/badssl.com/index.html]: We could either put this under ["Mixed Content"](https://badssl.com/#mixed-content) or we could rename ["HTTP Input"](https://badssl.com/#http-input) to something like "Non-secure input" and put it there. @marumari, @ericlaw1979: preference?

File: domains/mixed/mixed-form/index.html (hunk 1 above)
Comment (ericlaw1979) [inline domains/mixed/mixed-form/index.html]: done

File: domains/misc/badssl.com/index.html (hunk 2 above)
Comment (ericlaw1979) [inline domains/misc/badssl.com/index.html]: I'm inclined to leave it under "Mixed Content" as the UX is somewhat different. In the other cases, we're primarily testing "What happens when a HTTP page has a sensitive input" while in this case we're testing "What happens when a HTTPS page has a form with a non-secure target"

File: domains/mixed/mixed-form/index.html (hunk 1 above)
Comment (lgarron) [inline domains/mixed/mixed-form/index.html]: I still had to update the heading to match the other subdomains. I guess I wasn't clear, but just so you know: https://github.com/chromium/badssl.com/commit/8f7867b868e60e4805e4eb99ae1533fcdd55f73c

**Pull request merged**